cli/cli/helpers/portal_manager/portal_manager.go

```diff
@@ -213,6 +213,35 @@ func (portalManager *PortalManager) MapPorts(ctx context.Context, localPortToRem
 	return successfullyMappedPorts, failedPorts, nil
 }
 
+// DownloadAndStart downloads the required version and starts the portal if the required version
+// is not already running.
+func (portalManager *PortalManager) DownloadAndStart(ctx context.Context) error {
+	// Checking if new version is available and potentially downloading it
+	currentPortalPid, process, isPortalReachable, err := portalManager.CurrentStatus(ctx)
+	if err != nil {
+		return stacktrace.Propagate(err, "Unable to determine current state of Kurtosis Portal process")
+	}
+	if isPortalReachable {
```

Comment:
I might be missing something here, but I'm under the impression that if the portal is already running, we exit early and we don't even check whether it's running the required version. Whereas in the comment on top of this function you say "if the required version is not already running". 
It seems that we should validate the currently running process is the required version, and not an older one, right?